vpr/src/draw/draw.cpp

```diff
@@ -985,6 +985,7 @@ static void draw_router_expansion_costs(ezgl::renderer* g) {
 
 static void highlight_blocks(double x, double y) {
```

Comment:
1. Put a comment on what this function does in the .h file.  It highlights the block that was clicked on, looking from the top layer downwards for 3D devices (choose the top visible layer for overlapping blocks). It highlights the block, its fanin and fanout, in various colours by updating the draw_state variables.

2. Could consider factoring this routine to have a helper function that returns the blk_id (or -1) of the clicked on block.  That would shorten this function by moving the top part to another routine, and make commenting it easier.